modules/appeals_api/spec/requests/v2/notice_of_disagreements_controller_spec.rb

```diff
@@ -61,7 +61,7 @@ def new_base_path(path)
 
     context 'when a required headers is missing' do
       it 'returns an error' do
-        post(path, params: @data, headers: @headers.except('X-VA-File-Number'))
+        post(path, params: @minimum_data, headers: @headers.except('X-VA-File-Number'))
```

Comment:
Why these changes to use `@minimum_data`? It doesn't seem to make a material difference to the test itself - was it just easier to debug with smaller data?

Reply:
Because there was never an @data to begin with 😬 🤷‍♀️